docker/includes/security-headers-online.conf

```diff
@@ -3,4 +3,4 @@ include /etc/nginx/includes/security-headers-base.conf;
 # Govern what content can be loaded by the server and from where
 # Click jacking prevention to be used in addition to X-Frame-Options
 # Requires allowing inline-styles and inline data objects (svg imgs)
-add_header Content-Security-Policy "default-src 'self'; style-src 'self' 'unsafe-inline'; font-src 'self' data:; img-src 'self' data:; frame-ancestors 'self'; form-action 'self'; ";
+add_header Content-Security-Policy "default-src 'self'; style-src 'self' 'unsafe-inline'; font-src 'self' data:  https://cdn.jsdelivr.net/npm/monaco-editor@0.43.0/min/; img-src 'self' data:; frame-ancestors 'self'; form-action 'self'; script-src-elem 'self' https://cdn.jsdelivr.net/npm/monaco-editor@0.43.0/min/; style-src-elem 'self' 'unsafe-inline' https://cdn.jsdelivr.net/npm/monaco-editor@0.43.0/min/;";
```

Comment:
Bad news is that now monaco-editor for hawtio-react 1.5.0 is 0.52.0. Doesn't it mean we should use 0.52.0?
```console
$ yarn why monaco-editor
├─ @hawtio/react@npm:1.5.0
│  └─ monaco-editor@npm:0.52.0 (via npm:^0.52.0)
│
└─ @hawtio/react@npm:1.5.0 [1e960]
   └─ monaco-editor@npm:0.52.0 (via npm:^0.52.0)
```
Maybe we need to update Hawtio standalone as well... Hawtio 4.2.0 should be broken.

Reply:
let me double-check this. But AFAIR  that loader from 0.43 was used also in patternfly and  0.5 monaco. But I have to double-check

Reply:
@tadayosi ok, so patternfly-code-editor has dependency on the monaco-editor/react : https://github.com/patternfly/patternfly-react/blob/a3ffb39a0cc2c9130f7db86ad3186787ba12648e/packages/react-code-editor/package.json#L33 which has dependency on the monaco-editor/loader: https://github.com/suren-atoyan/monaco-react/blob/f7cac39fbad0f062dc66458831aaf57a7126dd40/package.json#L62 which contains: https://github.com/suren-atoyan/monaco-loader/blob/v1.4.0/src/config/index.js link to the version: `https://cdn.jsdelivr.net/npm/monaco-editor@0.43.0/min/` 

so the link is correct. @grgrzybek  Only question remains if we can replace this with "local" script. This is actually valid for both online and standalone.